Comments/ClientApp/src/styles/variables.scss

```diff
@@ -6,16 +9,37 @@
 }
 
 %all-drop-shadow {
-  box-shadow: 0 0 rem(10px) rgba(0, 0, 0, 0.1);
+  box-shadow: 0 0 utils.rem(10px) rgba(0, 0, 0, 0.1);
 }
 
 %transition-default {
   transition: all 0.3s ease-in-out;
   transition-property: transform;
 }
 
-$colour-agile-phase: #007AB8;
+%unstyled-button {
+  border: none;
+  background: transparent;
+  font-size: inherit;
+  padding: 0;
+}
+
+//$colour-agile-phase: #007AB8;
 
 $colour-caution-our-version: #FFBF47;
 
+$ec-white: global.$white;
+$ec-black: global.$black;
+
+$ec-light-grey: global.$custom-grey-4;
+$ec-dark-grey: global.$black-tint-1;
+$ec-green: #557021; //global.$custom-green-1
```

Comment:
There are some custom green/orange/pink values in the latest NDS alpha, so you might want to replace these hexes with those if they're suitable - I'm not sure where ec-green/ec-orange/ec-pink are used though, so if it's going to cause contrast issues etc. (or just generally make the page look awful) then by all means ignore me

Reply:
I'd dug out the nearest colours from globals, but not sure what the team want to do - use the new ones or retain the old ones. Should obviously be using the ds colours where possible, but think some of the old colours weren't replaced. I'll discuss it with the team, but think it's likely to go past the Wednesday deadline, so think this will suffice in the mean time.